docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
+
+## Adding Options
```

Comment:
I find this title confusing. I am guessing that you mean to talk about the possible ways to add a Visit or Timepoint type in LORIS?

Reply:
agreed - although not as concise, **Options for Adding Timepoints** would be much clearer

Reply:
@cmadjar what I mean by "Adding options" is: 
how to add a site, add a subproject, add... whatever. So adding site options, adding visit options... If you can think of other titles let me know, I'll change it for all the other docs

I would like to have the same title in all documents (ie not have add Visits, add subprojects, ...)
alternatives:
 - Adding to Database
 - How to Add?
 - Creating New
 - ????